pkgs/top-level/all-packages.nix

```diff
@@ -23642,9 +23642,7 @@ with pkgs;
     else if stdenv.hostPlatform.system == "riscv32-linux" then llvmPackages_14.libunwind
     else callPackage ../development/libraries/libunwind { };
 
-  libuv = callPackage ../development/libraries/libuv {
-    inherit (darwin.apple_sdk.frameworks) ApplicationServices CoreServices;
-  };
+  libuv = darwin.apple_sdk_11_0.callPackage ../development/libraries/libuv { };
```

Comment:
this doesn't affect linux builds I suppose ?

Reply:
yes, it shouldn't affect the Linux builds